registration/registration-services/src/main/java/io/mosip/registration/util/acktemplate/TemplateGenerator.java

```diff
@@ -520,6 +531,17 @@ private String getValue(Object fieldValue, String lang) {
 				|| fieldValue instanceof Double) {
 			value = String.valueOf(fieldValue);
 		}
+		
+		try {
+			List<String> blockListedWords = masterSyncServiceImpl.getAllBlackListedWords().stream().map(BlacklistedWordsDto::getWord).collect(Collectors.toList());
```

Comment:
instead we can fetch only words from DB. Please change query in the repository level

Reply:
Modified code as suggested.